src/views/PDFView.vue

```diff
@@ -48,17 +48,22 @@ export default {
 		},
 
 		isDownloadable() {
-			if (!this.file.shareAttributes) {
+			if (!this.file || !this.file.shareAttributes) {
```

Comment:
I would suggest to use instead the optional chanining operator:
```
if (!this.file?.shareAttributes) {
```
as it is shorter and also consistent with other usages (see, for example, [`isEditable()`](https://github.com/nextcloud/files_pdfviewer/blob/787fe745c9e7cd5c46a332a5bb1d9ec1cbc0ed31/src/views/PDFView.vue#L68-L70)).

The optional chaining operator should be applied too in `hideDownload` above.